test/org/sagebionetworks/bridge/TestUtils.java

```diff
@@ -538,12 +539,16 @@ public static JsonNode getOtherClientData() {
         return clientData;
     }
     
-    public static Set<String> getFieldNamesSet(JsonNode node) {
-        HashSet<String> set = new HashSet<>();
-        for (Iterator<String> i = node.fieldNames(); i.hasNext(); ) {
-            set.add(i.next());
+    public static void assertNode(JsonNode expected, JsonNode actual) {
+        assertEquals(expected.size(), actual.size());
+        
+        for (Iterator<Entry<String,JsonNode>> i = expected.fields(); i.hasNext();) {
+            Entry<String,JsonNode> expectedProperty = i.next();
```

Comment:
This is unnecessary. JsonNode.equals() does what you want it to do. Also, this method only verifies types, not values, doesn't work for non-object types, and doesn't do deep recursion.